x-pack/solutions/security/plugins/security_solution/public/onboarding/components/onboarding_body/cards/integrations_external_detections/integrations_card.tsx

```diff
@@ -29,18 +29,19 @@ const IntegrationsCardGridTabs: RenderChildrenType = ({
   checkCompleteMetadata = DEFAULT_CHECK_COMPLETE_METADATA,
   selectedTabResult,
 }) => {
-  const { available: list } = useEnhancedIntegrationCards(allowedIntegrations, {
+  const { activeIntegrations, isAgentRequired } = checkCompleteMetadata;
+
+  const { available: list } = useEnhancedIntegrationCards(allowedIntegrations, activeIntegrations, {
     showInstallationStatus: true,
     showCompressedInstallationStatus: true,
     returnPath: ONBOARDING_PATH,
   });
-  const { installedIntegrationsCount, isAgentRequired } = checkCompleteMetadata;
-
+  const activeIntegrationsCount = activeIntegrations?.length ?? 0;
```

Comment:
Is it possible to have undefined `activeIntegrations` if the `checkCompleteMetadata` has a default value?